### PR TITLE
naabu 2.0.2

### PR DESCRIPTION
**Pull request by alebcay**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/naabu.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Naabu < Formula
   desc "Fast port scanner"
   homepage "https://github.com/projectdiscovery/naabu"
-  url "https://github.com/projectdiscovery/naabu/archive/v1.1.4.tar.gz"
-  sha256 "fb5fae5ef7ef45217a238b59ed16095960c197c6f8b2f460873a445dbc80c2b4"
-  license "GPL-3.0"
```
Comment (chenrui333) [inline Formula/naabu.rb:6]: if we are not sure, we can just leave it as it is, I think.

generally, I think folks would still prefer `GPL-3.0-or-later` as compatibility choice, but i could be wrong though. :)

Comment (alebcay) [reply Formula/naabu.rb:6]: I agree that `GPL-3.0-or-later` would be more compatible, but `GPL-3.0-only` is the more conservative choice here. Open to other opinions on how to proceed here.

Comment (SMillerDev) [reply Formula/naabu.rb:6]: > generally, I think folks would still prefer GPL-3.0-or-later as compatibility choice, but i could be wrong though. :)

We're trying to represent the upstream license here though. Preference has no part in that. 

I'd just keep the identifier as it is and request clarification from upstream

Comment (chenrui333) [reply Formula/naabu.rb:6]: agree, that is what i was thinking.

**Comment by alebcay** [comment]
Upstream has re-tagged v2.0.0.

**Comment by alebcay** [comment]
https://github.com/projectdiscovery/naabu/issues/84

**Comment by SMillerDev** [comment]
Thanks @alebcay

File: Formula/naabu.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+  url "https://github.com/projectdiscovery/naabu/archive/v2.0.0.tar.gz"
+  sha256 "0d9e970c70b10d56695c8836847ee91a6717ceb1054e1ac7eb1d1c01437e0aa0"
+  license "GPL-3.0-only"
```
Comment (chenrui333) [inline Formula/naabu.rb:6]: if we can revert the license change, i think we can ship out the change.

Comment (chenrui333) [reply Formula/naabu.rb:6]: cc @Mzack9999

Comment (Mzack9999) [reply Formula/naabu.rb:6]: @chenrui333 The license has become MIT (https://github.com/projectdiscovery/naabu/blob/master/LICENSE.md)

**Comment by alebcay** [comment]
Leaving as do not merge for a bit because upstream seems to be tidying things up (including re-releasing tags, so CI here may not be accurate). Will re-visit in a day or two after things hopefully calm down.

**Comment by alebcay** [comment]
Hopefully things have settled down now.

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**